py/__init__.py

```diff
@@ -144,9 +162,11 @@ class PathHolders:
     "List_String": [PathHolders.ListMultiFrameCodec, PathHolders.StringCodec],
     "List_Data": [PathHolders.ListMultiFrameCodec, PathHolders.DataCodec],
     "ListCN_Data": [PathHolders.ListMultiFrameCodec, PathHolders.DataCodec],
+    "List_ListCN_Data": [PathHolders.ListMultiFrameCodec, PathHolders.ListCNDataCodec],
```

Comment:
Is this needed somewhere? I don't think this is used in SQL

Reply:
Good catch, removed it